src/zimdump.cpp

```diff
@@ -321,8 +321,8 @@ void ZimDumper::dumpFiles(const std::string& directory, bool symlinkdump, std::f
             write_to_file(directory + SEPARATOR, relative_path, blob.data(), blob.size());
 #else
             if (symlink(redirectPath.c_str(), full_path.c_str()) != 0) {
-              throw std::runtime_error(
-                std::string("Error creating symlink from ") + full_path + " to " + redirectPath);
+              //throw std::runtime_error(
+                std::string("Error creating symlink from ") + full_path + " to " + redirectPath;
```

Comment:
I'm not sure about this one.
Here we are failing to create a symlink. It can fails for a lot of reason. And even if we fail about a too long names, we must go through the error handling (and here, potentially print a warning only instead of failing if name is too long).